core/src/main/resources/hudson/model/View/popup.jelly

```diff
@@ -0,0 +1,43 @@
+<div class="remodal" data-remodal-id="modal">
```

Comment:
Are there suggestions on what can I do to include this popup.jelly file to sidepanel.jelly file? Not just inserting the remodal div into sidepanel.jelly as it is now. As discussed [here](https://github.com/samatdav/jenkins/commit/a82ebdbb31063604ad4b6e043232dc632b2068e9) with Tom, using <st:include page="popup.jelly"  /> does not quite work and throws an [error loading popup.jelly](https://gist.github.com/samatdav/4c51a0787eeda12460750520e7974245). The popup.jelly is placed in the same folder as sidepanel.jelly